### ISSUE TITLE
mouseup and mousedown API Event problem

### ISSUE THREAD

**Issue by designcomputer** [mpetroff/pannellum#749]
Hi mpetroff;

Just started looking at pannellum and I'm loving it so far. My problem is that I can't seem to capture mouse events. I've looked at some earlier examples like the one below from "How to get coordinates #243" , but still haven't had any luck getting this to work. Just FYI I've had no problem with the "load" or "scenechange" events.

viewer = pannellum.viewer(...);
viewer.on('mousedown', function(event) {
    // For pitch and yaw of center of viewer
    console.log(viewer.getPitch(), viewer.getYaw());
    // For pitch and yaw of mouse location
    console.log(viewer.mouseEventToCoords(event));
});

**Comment by mpetroff** [comment]
Which version of Pannellum are you using, and which browser / browser version? I just tested the `mousedown` / `mouseup` events (using the event handler you provided) with the Git `master` branch of Pannellum in Firefox 67 and Chromium 75, and it worked fine.

**Comment by designcomputer** [comment]
I am using Pannellum 2.4.0 with Chrome Version 75.0.3770.100 (Official Build) (64-bit) on Windows 10

**Comment by mpetroff** [comment]
I tested on Windows 10 with the same Chrome build and Pannellum version now, but I was still unable to reproduce the issue. Do you have an example that displays the issue that you're willing to share?

**Comment by designcomputer** [comment]
Sure, here is an example from the post I references, "How to get coordinates #243", that looks like it should work. https://frogfancy.com/test/

**Comment by mpetroff** [comment]
I'm really at a loss as to why it's not working for you. It works fine with every browser / computer I've tested it on. I've seen device-specific issues with WebGL before, but nothing like this with plain JavaScript. Could it be related to an add-on you're using?

![Screenshot from 2019-07-05 21-24-26](https://user-images.githubusercontent.com/1450212/60749689-7bde6400-9f6b-11e9-93dd-ff1b52d04e6c.png)

**Comment by designcomputer** [comment]
Interesting. The funny thing is some events fire like "load" and "scenechange". I'll test with addons off, on some other computers and with some other browsers.

**Comment by designcomputer** [comment]
I've turned off all my extensions in Chrome and debug mode too. Still no joy there. However, IE 11 (something) works fine. I also downloaded Chromium Version 77.0.3845.0 (Developer Build) (64-bit) and it works fine too. Now I am even more curious than when I though I just wasn't seeing something in the code. Sorry to bother you with this.

**Comment by designcomputer** [comment]
Here is what I've found. I usually work with the Chrome developer tools open but on my copy of Chrome Version 75.0.3770.100 (Official Build) (64-bit) when the developer tools are open the mouse events are being blocked somehow. If I close the developer tools and click on the Pannellum viewer the mouse events are captured and written to the log.  When I then reopen the developer tools I can see that they happened while it was closed. Very strange.
